src/mixins/I13nUtils.js

```diff
@@ -86,9 +94,10 @@ var I13nUtils = {
      * @return {Object} parent i13n node
      */
     _getParentI13nNode: function () {
-        // https://twitter.com/andreypopp/status/578974316483608576, get the context from parent context
+        var reactI13n = this._getReactI13n();
         var context = this.context;
-        return (context && context.i13n && context.i13n.parentI13nNode) || this._getReactI13n().getRootI13nNode();
+        return (context && context.i13n && context.i13n.parentI13nNode) || 
+            (reactI13n && reactI13n.getRootI13nNode());
```

Comment:
ok yeah, as we discussed, it's `context.i13n.parentI13nNode` shouldn't be access by users directly, what they will be used is `context.i13n.getI13nNode().getMergedModel()`, this function travers back to the root and find merged model, we handle there already https://github.com/yahoo/react-i13n/blob/master/src/libs/I13nNode.js#L103